.github/workflows/cd_prod.yml

```
@@ -0,0 +1,101 @@
+    name: Deploy iOS app
+    permissions:
+      contents: read
+    runs-on: macos-14
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
+        with:
+          fetch-depth: 0
+      - name: Setup Flutter
+        uses: ./.github/actions/setup-flutter
+      - name: Setup Xcode
+        uses: ./.github/actions/setup-xcode
```

actionlint

name is required in action metadata "/cache/22/52c4aa01-d1ca-40d1-a517-bc8321ba27c0/home/jailuser/git/.github/actions/setup-xcode/action.yml"

(action)